numpy/lib/stride_tricks.py

```diff
@@ -179,7 +179,7 @@ def _broadcast_shape(*args):
     supplied arrays against each other.
     """
     if not args:
-        raise ValueError('must provide at least one argument')
+        return ()
     # use the old-iterator because np.nditer does not handle size 0 arrays
     # consistently
     b = np.broadcast(*args[:32])
```

Comment:
@mhvk: So you'd be in favor of a separate PR that removes that restriction from the C code?